packages/lambdas/src/__tests__/document-downloader.test.ignore.ts

```diff
@@ -10,10 +10,11 @@ const cwOrgCertificate = ``;
 const cwOrgPrivateKey = ``;
 const bucketName = ``;
 
-const file = {
+const docRef = {
```

Comment:
NABD, but maybe we can add a JSDoc here indicating that these should be updated before running the tests